db.go

```
@@ -705,81 +693,91 @@ func (a *Db) FetchSlaveWithContext() (*sql.Rows, error) {
 
 func (a *Db) SetParams(params ...interface{}) {
-
 	a.Params = params
 }
 
 func (a *Db) SetQuery(query string) {
-
 	a.Query = query
 }
 
 func (a *Db) setResults(result ...interface{}) {
```

golangci-lint (2.12.2)

[error] func (*Db).setResults is unused

(unused)